app/composables/dashboard/useDcBaseChart.ts

```
@@ -0,0 +1,168 @@
+    clearTimeout(pressTimer as ReturnType<typeof setTimeout>);
+    pressTimer = setTimeout(() => {
+      isMouseLongClick.value = true;
+    }, mouseLongClickDuration);
+  };
+
+  const handleMouseUp = (event: MouseEvent | TouchEvent) => {
+    clearTimeout(pressTimer as ReturnType<typeof setTimeout>);
+  };
+
+  // フィルタ追加ハンドラ - 単一選択と追加選択を処理
+  const addFilterHandler = (filters: any[], filter: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)